### DISCUSSION TITLE
Warning: Add event_register()

### DISCUSSION THREAD

**Discussion by jeanneyue** [insightsengineering/teal#1251] [Q&A] [answered]
Hi,

I tried out some apps from the gallery locally. I got the following the warnings:

Warning: The 'plotly_relayout' event tied a source ID of 'xxx' is not registered. In order to obtain this event data, please add `event_register(p, 'plotly_relayout')` to the plot (`p`) that you wish to obtain event data from.

May I know how to handle it?

Best,
Jeanne

**Comment by donyunardi** [comment]
I believe it's coming from filter panel (teal.slice) for numeric ranges filter when building the histogram visualization when the card is opened. 

@chlebowa do you recall seeing this during the development of https://github.com/insightsengineering/teal.slice/pull/289?

So far, it feels quite harmless.

**Reply by chlebowa** [reply, marked as answer]
Some years ago the consensus was these warnings cannot be silenced. Following the instructions and properly regisetring the events does (did) nothing. They are harmless, if annoying.

**Reply by donyunardi** [reply]
Thanks @chlebowa!

@jeanneyue it should be safe to ignore this. Let us know if it has some side effects on your end.

**Comment by jeanneyue** [comment]
Thanks a lot for the confirmation. The teal examples worked.
